code/game/turfs/simulated/floor/misc_floor.dm

```diff
@@ -265,3 +265,36 @@
 	for(var/obj/structure/spacevine/SV in src)
 		if(!QDESTROYING(SV))//Helps avoid recursive loops
 			qdel(SV)
+
+/turf/open/floor/plasteel/acid_spray
+	floor_tile = /obj/item/stack/tile/acid_spray
+	var/spray_amount = 35
+	var/cooldown = 150
+	var/last_spray = 0
+	var/primed = FALSE
```

Comment:
I'm not sure whether having the functionality attached to the turf is the best. @ChangelingRain, this okay?